src/local-time.lisp

```diff
@@ -2015,3 +2015,37 @@ You can see examples in +ISO-8601-FORMAT+, +ASCTIME-FORMAT+, and +RFC-1123-FORMA
   (- (day-of timestamp) +modified-julian-date-offset+))
 
 (declaim (notinline format-timestring))
+
+(defun encode-universal-time-with-tz (sec minute hour day month year &key timezone)
```

Comment:
I intentionally didn't include this function in the library because I wanted users to know, explicitly, that doing this will encode a guess as to which subzone is being used, and that being a guess means that it could be incorrect.